src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -493,27 +520,35 @@ test_budgets_multi(void)
 
 	PRINTC("Starting multi-level budget test.\n");
 
-	exec_cluster_alloc(&mbt.p, spinner_cyc,  &(mbt.p.cyc), BOOT_CAPTBL_SELF_INITRCV_BASE);
+	exec_cluster_alloc(&mbt.p, spinner_cyc, &(mbt.p.cyc), BOOT_CAPTBL_SELF_INITRCV_BASE);
 	exec_cluster_alloc(&mbt.c, spinner_cyc, &(mbt.c.cyc), mbt.p.rc);
 	exec_cluster_alloc(&mbt.g, spinner_cyc, &(mbt.g.cyc), mbt.c.rc);
 
 	PRINTC("Budget switch latencies:\n");
-	for (i = 1 ; i < 10 ; i++) {
+	for (i = 1; i < 10; i++) {
 		tcap_res_t res;
-		thdid_t    tid;
-		int        blocked;
-		cycles_t   cycles, s, e;
+		thdid_t tid;
+		int blocked;
+		cycles_t cycles, s, e;
 
 		/* test both increasing budgets and constant budgets */
-		if (i > 5) res = 1600000;
-		else       res = i * 800000;
+		if (i > 5)
```

Comment:
in addition to UR, it should know that aside from same-line bodies, it requires `{}`